src/styles/theme-manager.js

```diff
@@ -191,8 +191,35 @@ module.exports = {
         backgroundColor: 'transparent',
         borderColor: rawTheme.palette.borderColor,
       },
+      isRtl: rawTheme.isRtl,
```

Comment:
For now it's just "linear-progress". It can be easily removed though and then we wouldn't need those constant values in theme.